user-docs/latest/content-manager/configuring-view-of-content-type.md

```diff
@@ -0,0 +1,41 @@
+# Configuring the view of a content type
```

Comment:
Is it normal that this title is not the same as the one in the navigation on the left ("Configuring a content type view")?

Reply:
Good catch!